Tasks/Ann_Valenda/task5/ex1-3py

```diff
@@ -0,0 +1,71 @@
+import re
```

Comment:
first of all, it's not a python file, why??

secondly, can you do anything without regular expressions?))